workers/data_refinery_workers/processors/smasher.py

```diff
@@ -485,22 +493,25 @@ def _smash(job_context: Dict, how="inner") -> Dict:
 
         # Smash all of the sample sets
         logger.debug("About to smash!",
-                     input_files=job_context['input_files'],
-                     dataset_data=job_context['dataset'].data,
+                     input_files=job_context['input_files'].keys(),
```

Comment:
`job_context['input_files']` is a list of tuples. I don't think `.keys()` should work.

Reply:
Nvm, it's not! It's a dict! My bad!